test/relay_client_test.js

```diff
@@ -365,7 +365,7 @@ contract('RelayClient', function (accounts) {
         let fromAddr = ephemeralKeypair.address
         rc.useKeypairForSigning(ephemeralKeypair)
         var did_assert = false
-        rc.sendViaRelay = function(relayUrl, signature, from, to, encodedFunction, gasprice, gaslimit, relayFee, nonce, relayHubAddress, relayAddress){
+        rc.sendViaRelay = function(relayUrl, signature, approvalData, from, to, encodedFunction, gasprice, gaslimit, relayFee, nonce, relayHubAddress, relayAddress){
```

Comment:
The idea was to move ALL occurrences of `signature` and  `approvalData` : they both should be at the end, after all signed parameters, and `approvalData` the last param.

Reply:
Right, I though Alex simply meant having the order of signature and approvalData be consistent. If you look at `sendViaRelay` for example, many other values are out of order: the relay fee should go before the gas price, but it is after the gas limit. Should I also fix those here?

Reply:
Yes, these should be fixed too. (there's also a small issue of the CanRelay in the go code,  which uses different parameter ordering, and can be fixed too)